src/libraries/System.Diagnostics.Process/src/System/Diagnostics/Process.Unix.cs

```diff
@@ -26,6 +26,7 @@ public partial class Process : IDisposable
         /// Puts a Process component in state to interact with operating system processes that run in a
         /// special mode by enabling the native property SeDebugPrivilege on the current thread.
         /// </summary>
+        [System.Runtime.Versioning.MinimumOSPlatformAttribute("windows7.0")]
```

Comment:
Does this actually need to be annotated?  Same question for LeaveDebugMode.  It probably doesn't matter as basically no one uses it, but it's also unlikely to be a problem if it's in an app being brought over to Unix; if this is an issue, it's almost certain there are much bigger problems.